input/resources/codesystem-au-recorded-sex-or-gender-type.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="au-recorded-sex-or-gender-type" />
+    <meta>
+        <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem" />
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+        <valueInteger value="0"/>
+    </extension>
+    <url value="http://terminology.hl7.org.au/CodeSystem/au-recorded-sex-or-gender-type" />
+    <identifier>
+        <system value="urn:ietf:rfc:3986"/>
+        <value value="urn:oid:2.16.840.1.113883.2.3.4.1.4.29"/>
+    </identifier>
+    <name value="AURecordedSexOrGenderType" />
+    <title value="AU Recorded Sex or Gender Type" />
+    <status value="active" />
+    <experimental value="false"/>
+    <description value="Codes that represent the type of a Recorded Sex or Gender (RSG) value in an Australian context." />
```

Comment:
Change the prelim text we had in the draft files "Codes that represent the type of a Recorded Sex or Gender (RSG) value in an Australian context."

To be something like "The AU Recorded Sex or Gender Type code system defines concepts that represent the type or category of sex or gender that is recorded."

Reply:
Done.